web/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,434 @@
+//@ts-nocheck
```

Comment:
Would be great if only the needed checks (i.e. config missing) could be disabled.

Reply:
I can use @ts-ignore but have to use it on all lines that uses config or similar and i guess that would be worse